src/core/3d/qgsabstract3dsymbol.h

```diff
@@ -43,8 +43,9 @@ namespace Qt3DCore { class QEntity; } SIP_SKIP
  *
  * \since QGIS 3.0
  */
-class CORE_EXPORT QgsAbstract3DSymbol
+class CORE_EXPORT QgsAbstract3DSymbol : public QObject
```

Comment:
same here -- there's no reason to make this a qobject (and a ton of reasons not to)